frontend/ad-mission/src/components/Payment.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="form-container">
+    <h1 class="form-title">Ad Mission</h1>
+    <form @submit.prevent="submitForm">
+      <div>
+        <label for="cardNumber">Card Number:</label>
+        <input type="text" id="cardNumber" v-model="cardNumber" required>
+      </div>
+      <div>
+        <label for="expiry">Expiry Date:</label>
+        <input type="text" id="expiry" placeholder="MM/YYYY" v-model="expiry" required>
+      </div>
+      <div>
+        <label for="cvv">CVV:</label>
+        <input type="text" id="cvv" v-model="cvv" required>
+      </div>
+      <div>
+        <label for="name">Cardholder Name:</label>
+        <input type="text" id="name" v-model="cardholderName" required>
+      </div>
+      <button type="submit">Submit Payment</button>
+    </form>
+  </div>
+</template>
+
+
+<script>
+export default {
+  data() {
+    return {
+      cardNumber: '',
+      expiry: '',
+      cvv: '',
+      cardholderName: ''
+    };
+  },
+  methods: {
+    submitForm() {
+      // Input validation
+      if (!/^\d{16}$/.test(this.cardNumber)) {
+        alert('Please enter a valid 16-digit card number.');
+        return;
+      }
+
+      if (!/\d{2}\/\d{4}/.test(this.expiry)) {
+        alert('Please enter a valid expiry date in MM/YYYY format.');
+        return;
+      }
+
+      if (!/^\d{3}$/.test(this.cvv)) {
+        alert('Please enter a valid 3-digit CVV.');
+        return;
+      }
+
+      // Form submission logic
+      console.log('Form submitted with data:', {
+        cardNumber: this.cardNumber,
+        expiry: this.expiry,
+        cvv: this.cvv,
+        cardholderName: this.cardholderName
+      });
+
+      // Reset form fields after submission if needed
+      this.cardNumber = '';
+      this.expiry = '';
+      this.cvv = '';
+      this.cardholderName = '';
+    }
+  }
+};
+</script>
+
+
+<style scoped>
```

Comment:
Make sure to properly style ur page. As of now this page is only covering small section in middle of screen.